lib/_http_outgoing.js

```diff
@@ -121,6 +122,8 @@ function OutgoingMessage() {
 ObjectSetPrototypeOf(OutgoingMessage.prototype, Stream.prototype);
 ObjectSetPrototypeOf(OutgoingMessage, Stream);
 
+OutgoingMessage.prototype[SymbolAsyncIterator] = null;
```

Comment:
I would prefer we had a generic way for detecting and doing this.

Reply:
Any suggestions? The only way I can think of is using `readable`/`writable` but those can change... i.e. we might start as `!readable` and later become `readable` (not sure how these are currently used in the wild).

Reply:
We can make `Symbol.asyncIterator` a getter on `Stream`, checking if read exists and return null otherwise.

Reply:
But the point here is to support v1 streams that do not have `read`?

Reply:
Sorry, I meant _resume_ : https://github.com/nodejs/node/blob/f4a4a1a63c7ceba77a0178b4af6068abcadba41f/lib/internal/streams/legacy.js#L29.

On second thought I'm not convinced we should add this to the Stream1 prototype after all. What's your use case?

Reply:
We have v1 streams that might be used in e.g. `pipeline` and `for await`... I would expect those to work?